spec/queries/my_facilities/blood_pressure_control_query_spec.rb

```diff
@@ -209,25 +246,25 @@
       end
 
       describe '#cohort_registrations' do
-        specify do
+        it 'counts registrations diagnosed with hypertension' do
```

Comment:
We can wrap the `specify` under a context of `with htn` or some such and leave the `specify` intact.